doc/releases/squid.rst

```diff
@@ -27,7 +29,8 @@ Changelog
 v19.2.1 Squid
 =============
 This is the first backport release in the Squid series.
-We recommend that all users update to this release.
+
+.. warning: Do not upgrade to Squid v19.2.1. Upgrade instead to Squid v19.2.1.
```

Comment:
I think you mean v19.2.2 at the end of this line. 

That said, I suggest that the two cogent warnings should be in the same order, as is it's kind of a red zone and white zone dissonance and slightly prone to the gentle reader glossing over the specifics.

So basically like 33 should be the same as line 12.

Reply:
You are correct.

Reply:
Thanks, Ed.